metadata-ingestion/src/datahub/cli/docker_cli.py

```diff
@@ -803,28 +858,44 @@ def quickstart(
     )
 
 
+def get_docker_compose_base_url(version_tag: Optional[str]) -> str:
+    if os.environ.get("DOCKER_COMPOSE_BASE"):
```

Comment:
@pedro93 you can use the `DOCKER_COMPOSE_BASE`

Reply:
Missed this, was still going through the PR. Thanks!